eks_fargate/README.md

```diff
@@ -124,7 +124,178 @@ The setup below configures the Cluster Agent to communicate with the Agent sidec
+
+   ```yaml
+    apiVersion: datadoghq.com/v2alpha1
+    kind: DatadogAgent
+    metadata:
+      name: datadog
+    spec:
+      global:
+        clusterAgentTokenSecret:
+          secretName: datadog-secret
+          keyName: token
+        credentials:
+          apiSecret:
+            secretName: datadog-secret
+            keyName: api-key
+      features:
+        admissionController:
+          agentSidecarInjection:
+            enabled: true
+            provider: fargate
+   ```
+  Then apply the new configuration:
+  
+  ```shell
+  kubectl apply -n datadog-agent -f datadog-agent.yaml
+  ```
+
+2. After the Cluster Agent reaches a running state and registers Admission Controller mutating webhooks, an Agent sidecar is automatically injected into any pod created with the label `agent.datadoghq.com/sidecar:fargate`. 
+   **The Admission Controller does not mutate pods that are already created**.
+
+**Example result**
+
+The following is a `spec.containers` snippet from a Redis deployment where the Admission Controller injected an Agent sidecar. The sidecar is automatically configured using internal defaults, with additional settings to run in an EKS Fargate environment. The sidecar uses the image repository and tags set in the Helm values. Communication between Cluster Agent and sidecars is enabled by default. 
+
+{{< highlight yaml "hl_lines=7-29" >}}
+  containers:
+  - args:
+    - redis-server
+    image: redis:latest
+  # ...
+  - env:
+    - name: DD_API_KEY
+      valueFrom:
+        secretKeyRef:
+          key: api-key
+          name: datadog-secret
+    - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+      valueFrom:
+        secretKeyRef:
+          key: token
+          name: datadog-secret
+    - name: DD_EKS_FARGATE
+      value: "true"
+    # ...
+    image: gcr.io/datadoghq/agent:7.51.0
+    imagePullPolicy: IfNotPresent
+    name: datadog-agent-injected
+    resources:
+      limits:
+        cpu: 200m
+        memory: 256Mi
+      requests:
+        cpu: 200m
+        memory: 256Mi
+{{< /highlight >}}
+
+###### Sidecar profiles and custom selectors
+
+To further configure the Agent or its container resources, use the properties in your `DatadogAgent` resource. Use the `spec.features.admissionController.agentSidecarInjection.profiles` to add environment variable definitions and resource settings. Use the `spec.features.admissionController.agentSidecarInjection.selectors` property to configure a custom selector to target workload pods instead of updating the workload to add `agent.datadoghq.com/sidecar:fargate` labels.
+
+  1. Create a `DatadogAgent` custom resource in `datadog-values.yaml` file that configures a sidecar profile and a custom pod selector.
+
+   **Example**
+
+   In the following example, a selector targets all pods with the label `"app": redis`. The sidecar profile configures a `DD_PROCESS_AGENT_PROCESS_COLLECTION_ENABLED` environment variable and resource settings. 
+
+   ```yaml
```

Comment:
Can you indent this example section (lines 212 to 240) by two spaces?

Reply:
I've adjusted the indentation. Could you please review? :)

Reply:
https://github.com/DataDog/integrations-core/commit/601742ac609f9045da707e3ed1d2dacf65a0a647